media-sound/picard/picard-2.1.3.recipe

```diff
@@ -4,35 +4,36 @@ HOMEPAGE="https://picard.musicbrainz.org/"
 COPYRIGHT="2004-2019 Robert Kaye, Lukas Lalinsky, Laurent Monin, \
 Sambhav Kothari, Philipp Wolfer and others"
 LICENSE="GNU GPL v2"
-REVISION="4"
+REVISION="5"
 SOURCE_URI="ftp://ftp.eu.metabrainz.org/pub/musicbrainz/picard/picard-$portVersion.tar.gz"
 CHECKSUM_SHA256="8e044fe68c44d345c19f98952c3c7f20c72da74dbac9ce7c7b0621e2d69885a7"
 SOURCE_DIR="picard-release-$portVersion"
 PATCHES="picard-$portVersion.patchset"
 ADDITIONAL_FILES="picard.rdef.in"
 
-ARCHITECTURES="x86 x86_64"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
 
 PROVIDES="
-	picard = $portVersion
+	picard${secondaryArchSuffix} = $portVersion
 	cmd:picard = $portVersion
 	"
 REQUIRES="
-	haiku
+	haiku${secondaryArchSuffix}
 	cmd:python3.6
-	pyqt_python3
+	pyqt_python3${secondaryArchSuffix}
```

Comment:
just pyqt_python3 will do (no need for secondaryArchSuffix)
PS we only use the brackets for a few things :)